frontend/src/components/charts/MoneyVsPlaytimeDoughnut.tsx

```diff
@@ -0,0 +1,47 @@
+import {UserGameDTO} from "../../service/model";
+import {Doughnut} from "react-chartjs-2"
+import {Chart, ArcElement, Legend, Tooltip, Title} from "chart.js"
+Chart.register(ArcElement, Legend, Tooltip, Title);
+
+interface gameDoughnutProps{
+    Game: UserGameDTO
```

Comment:
would probably make variable names lowerCase. Shouldn't be a problem here but always seems like a class name

Reply:
oh yes, didn't know why i write it in upperCase here, i will change it 👍